### ISSUE TITLE
confirmTransportationOptions frequently fails with InternalServerError for multi-shipment placement

### ISSUE THREAD

**Issue by johnkw** [amzn/selling-partner-api-models#4436]
The `https://sellingpartnerapi-na.amazon.com/inbound/fba/2024-03-20/operations/...` call fails with:

```{"operationStatus":"FAILED","operationId":"[...]","operation":"confirmTransportationOptions","operationProblems":[{"severity":"ERROR","code":"InternalServerError","message":"ERROR: Something went wrong. Please try again later."}]}```

This is essentially a duplicate of bug #4144 but that one was closed without any resolution or workarounds being posted.  It was also mentioned in bug #4133 and in https://github.com/amzn/selling-partner-api-models/issues/3951#issuecomment-2323628180

Deleting the entire Workflow and trying again doesn't seem to help at all, it just keeps happening.

**Comment by johnkw** [comment]
After several Workflow cancel/delete and start over, it eventually worked. So it doesn't always fail, but it fails quite a lot.

**Comment by LinyuPu-pacvue** [comment]
I also have the same problem with single shipment, whenever i chosen the UPS

**Comment by LennyPu** [comment]
I also have the same problem with single shipment, whenever i chosen the UPS

**Comment by puppsupr** [comment]
Hi @johnkw, @LennyPu,
In order to correctly resolve this issue, Please [open a support case](https://developer.amazonservices.com/support) following the [guidance](https://developer.amazonservices.com/support-guidelines) here so we can pursue the investigation as additional details are required from your end.

**Comment by johnkw** [comment]
Well I did already open a support case days ago but there's been no response.  The important thing for anyone that finds this ticket to know is: keep deleting the entirely workflow and trying again. It seems it's possible to eventually get it to work.

**Comment by puppsupr** [comment]
Hi @johnkw,
Can you provide the case number ?

**Comment by johnkw** [comment]
16853613761

**Comment by lightningja** [comment]
second that. created several support cases but you kept asking for more info without actually investigating it. I've provided the request Id, inbound plan Id and operation Id.

**Comment by puppsupr** [comment]
Hi @johnkw, the case 16853613761 is being actively investigated, you see more details over the case once the investigation is done.

@lightningja, can you please provide your case number as well ?

**Comment by luap2703** [comment]
This is ridiculous!!! How is anyone suppost to use this sh*tty API if there are around 20 open issues just for creating a single inbound shipment?!?!

Opening a support ticket doesn't help, opening a github ticket gets closed as noone in this 200k people organisation feels responsible...

GET THIS FIXED!

**Comment by johnkw** [comment]
This was at least mostly fixed at some point. We haven't seen this bug in 9 months or so.
